quinn/src/tests.rs

```diff
@@ -387,7 +387,10 @@ async fn zero_rtt() {
 }
 
 #[test]
-#[cfg_attr(target_os = "solaris", ignore = "Fails on Solaris")]
+#[cfg_attr(
+    any(target_os = "solaris", target_os = "illumos"),
```

Comment:
Nit: could we use `solarish` here, too?

Reply:
Friendly ping @AsakuraMizu.

Reply:
I suggest doing this in a separate pull request since the `quinn` crate currently does not have a custom build script or cfg aliases setup.